doc/gallery-src/analysis/run_hexReactorToRZ.py

```diff
@@ -0,0 +1,40 @@
+"""
+Hex reactor to RZ geometry conversion
+===================================
+This shows how an entire reactor specified in full hex detail can be
+automatically converted to an equivalent 2-D or 3-D RZ case.
```

Comment:
"equivalent" might be a bit strong. "representative?"

Reply:
updated to say conserved mass